declipping/declip-simple.py

```diff
@@ -0,0 +1,111 @@
+"""This function takes one clipped file and declips it. It was created for
+the declipping blog post."""
+from scipy.io.wavfile import read, write
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+from scipy.interpolate import interp1d
+
+def get_segments(np_array):
+    nmax = max(np_array)
+    nmin = min(np_array)
+
+    clipped_segments = []
+    inside_clip = False
+    clip_start = 0
+    clip_end = 0
+
+    for i, sample in enumerate(np_array):
```

Comment:
Are you looking for this function here?

https://docs.scipy.org/doc/numpy/reference/generated/numpy.where.html

Reply:
Heya this is great thanks for sharing! Though more efficient, I think it would be a bit more confusing for the tutorial, because I break these lines down and explain them in chunks. You can see that here: But gtk that this exists for the future!